### DISCUSSION TITLE
Self hosted Zitadel in Kubernetes with managed Postgres

### DISCUSSION THREAD

**Discussion by Oscaarjs** [zitadel/zitadel#8365] [Q&A]
Hello!

I'm trying to deploy Zitadel in our kubernetes cluster (GKE) using a managed postgres instance (google cloud sql). The way I typically connect to my DBs from the cluster is by using a sidecar container running the cloud sql proxy. As far as I can see there's no configs that allow this without modifying the actual chart. 

**What's the recommended way of connecting to a managed DB like this?** I find the documentation for the charts rather slim. What I've tried so far is to have the zitadel as a subchart of "my-zitadel" (umberella chart). Here I'm also deploying a cloud sql proxy and a service for it so that I can connect to the db using this service. However this seems to result in a bit of a "chicken & egg problem" as the init & setup jobs need to connect to the DB, but the sql proxy deployment doesn't seem to start before k8s jobs.

**Comment by hifabienne** [comment]
@eliobischof can you help here?

**Comment by eliobischof** [comment]
Would this help? https://github.com/zitadel/zitadel-charts/issues/207

**Reply by Oscaarjs** [reply]
I would assume that could help yes, but do I understand it correctly that this is in the scope of features to be developed but not yet done? If so, are there any other recommended ways of solving this "issue"? Best

**Comment by fforootd** [comment]
Just as backlink, relates to https://discord.com/channels/927474939156643850/1019895518383583323/threads/1267789728560844830
